Jenkinsfile

```
@@ -382,6 +387,44 @@ node ('master') {
                          x64LinuxAlwaysPostSteps, "x86_64 Linux ${build_type} ${default_compiler} Burrow", x64LinuxWorker, tasks)
     }
   }
+
+  def s390xLinuxBuildSteps
+  def s390xLinuxPostSteps = new Builder.PostSteps()
+  if(!s390xlinux_compiler_list.isEmpty()){
+    s390xLinuxAlwaysPostSteps = new Builder.PostSteps(
+      always: [{x64LinuxBuildScript.alwaysPostSteps(scmVars, environmentList, coredumps)}])
+    s390xLinuxPostSteps = new Builder.PostSteps(
+      always: [{x64LinuxBuildScript.alwaysPostSteps(scmVars, environmentList, coredumps)}],
+      success: [{x64LinuxBuildScript.successPostSteps(scmVars, packagePush, pushDockerTag, environmentList)}])
+    def first_compiler = s390xlinux_compiler_list[0]
+    def release_build = specialBranch && build_type == 'Debug'
+    def manifest_push = false
+
+    // register first compiler with packageBuild, and manifest push
+    registerBuildSteps([{x64LinuxBuildScript.buildSteps(
+                       parallelism==0 ?s390xLinuxWorker.cpusAvailable : parallelism, first_compiler, build_type, build_shared_libs, specialBranch, coverage,
+                       testing, testList, false, false, false, false, packageBuild, sanitize, fuzzing, benchmarking, coredumps, useBTF, use_libursa, use_burrow,
```

```suggestion
                       testing, testList, /*cppcheck*/false, /*sonar*/false, false, false, packageBuild, sanitize, fuzzing, benchmarking, coredumps, useBTF, use_libursa, use_burrow,
```
Please add similar comments for other parameters as it is done for other build steps. Not the best approach in general, but it will be easier to read.